nimare/base.py

```diff
@@ -283,14 +287,35 @@ def __init__(self, *args, **kwargs):
             mask = get_masker(mask)
         self.masker = mask
 
+        self.resample = kwargs.get("resample", False)
+
+        # defaults for resampling images (nilearn's defaults do not work well)
+        self.resample_kwargs = kwargs.get(
+            "resample_kwargs", {"clip": True, "interpolation": "linear"}
+        )
```

Comment:
In this approach, we're talking about feeding in a single argument, `resample_kwargs`, as a dictionary. In order, similar situations in NiMARE, we instead accept individual arguments with a prefix instead of bundling them in a dictionary. See how we work with kernel transformer arguments fed into meta-analytic estimators:

https://github.com/neurostuff/NiMARE/blob/106b51634707fd7999617f277378aefc349867f8/nimare/meta/cbma/base.py#L41-L44

If you want to go with the dictionary-based approach, I'd like to propagate it across the package.

Reply:
I do not have strong feelings about the approach, I'll switch to prefixes for consistency.